functional-tests/wdio.conf.ts

```diff
@@ -0,0 +1,66 @@
+import { Config } from "webdriverio";
+
+const isInDocker = !!process.env.IN_DOCKER,
+	isTeamCity = !!process.env.TEAMCITY_VERSION;
+
+export const config: Config = {
```

Comment:
Are tests still running `sync=true` previously they were. Was there a reason for this? Parallel issues?

Reply:
Nope, sync has been removed. It's not an option in WDIO6 like it was in v4. Instead v6 looks for the @wdio/sync module being installed. We're doing everything async because it's The Node Way, and avoids having to use fibers, which is fraught with issues like needing binaries etc and is obsolete anyway.

Reply:
Ah Ok course cool.